lib/mongodb.js

```diff
@@ -816,18 +843,83 @@ MongoDB.prototype.all = function all(model, filter, options, callback) {
     query = self.buildWhere(model, filter.where);
   }
   var fields = filter.fields;
+  var relations = this._models[model].settings.relations;
```

Comment:
@shalkam I am not an expert of http(sorry lol) so need some time to check why we can get it through http, but this looks like a bug to me, I might be wrong but  IMO connector should not see its attached model's definitions....